python/tvm/topi/cuda/sort.py

```diff
@@ -201,9 +201,9 @@ def sort_nms_ir(data, valid_count, output, axis, is_ascend):
     nthread_tx = max_threads
     nthread_bx = size // max_threads + 1
     tx = te.thread_axis("threadIdx.x")
-    bx = te.thread_axis("vthread")
+    bx = te.thread_axis("blockIdx.x")
     ib.scope_attr(tx, "thread_extent", nthread_tx)
-    ib.scope_attr(bx, "virtual_thread", nthread_bx)
+    ib.scope_attr(bx, "thread_extent", nthread_bx)
```

Comment:
Could you leverage more on the changes of this file based on performance results?

Reply:
I have not been able to pass tests with the kernels currently in master, so I can't compare apples to apples